README.md

```diff
@@ -15,7 +15,7 @@ For this workshop, we'll try to have you implement a CLI client (command line in
 The rules are simple: The game is a duel between two players. It is played turn by turn and each turn, one player adds a token in the grid by selecting a column. The token then fall down to the end of the column. There is a winner when a player successfully align 4 tokens of his color in any direction (horizontal, vertical or diagonal).
 
 ## The Network
-For this workshop, we won't use directly the Ethereum Network but the Polygon one, which share exactly the same interface and has the same capatibilities, except it's a lot faster. We will use the Mumbai testnet to avoid spending real money :)
+For this workshop, we won't use directly the Ethereum Network but the network from the Polygon side-chain, which share exactly the same interface, except it's a lot faster due to the design choices made at the time of its conception. We will use the Mumbai testnet to avoid spending real money :)
```

Comment:
Yeah, I'm bullish on Polygon especially since they acquired the Hermez network but I think it's crucial to mention that the network is "faster" due to the design choices they made. I understand that it's not a good place to exposes differences between Ethereum networks, side-chains, L2 etc... But by reading the initial sentence I had the feeling that Polygon was better than Ethereum without any default which is not true, that's why I suggest a description more nuanced.